extras/parse_travis_log.py

```diff
@@ -899,7 +900,6 @@ def build_return_code(status_vera_init,
        (status_make_install) and \
        (status_tests) and \
        (number_of_msgs_in_summary(summary_vera) == 0) and \
-       (number_of_msgs_in_summary(summary_cppcheck) == 0) and \
```

Comment:
@gtrensch: you convinced me! I agree that we should rather fix the code instead of having an advanced system to disable checks. I also agree with @heplesser's suggestion to leave the line that disables cppcheck for now in as a comment with some explanatory text.